liquidator/test/common.js

```diff
@@ -0,0 +1,45 @@
+const Token = artifacts.require("ExpandedERC20");
+
+const { toBN } = web3.utils;
+
+const CONSTANTS = {
+  ETH_ADDRESS: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
+};
+
+const assertBNGreaterThan = (a, b) => {
```

Comment:
I'm pretty sure you can do this directly with the `gt` & `gte` directly within the bignumbers? why does it need a separate util for this action?

Reply:
Gte and gt on chai's assert (what truffle is using) (from memory) doesnt support big integers that well. And you're right that we can just do

`assert.equal(bnA.gte(bnB))` but that wouldnt provide enough information during debugging which  is why this function exists

Reply:
ok cool but this is different from how we do this in other places in the repo though. Also you already have this function in the `test-fork` director [here](https://github.com/UMAprotocol/protocol/blob/0fe67182ecc811c8981b901856b1ca5ff6dc6f63/liquidator/test-fork/OneInchExchange.js#L12). If we want to have a specific function to do this then should it not be put in the common directory or more generalized test helpers? I'm just trying to think how this would play out once we need to re-use this logic elsewhere and then it's in this package.